conference/src/components/CreateSubmissionForm/CreateSubmissionForm.tsx

```diff
@@ -87,24 +87,120 @@ export const CreateSubmissionForm = ({ submission }: { submission?: Submission |
+                <ClearIcon />
+              </IconButton>
+            </Box>
+          }
+          <Button fullWidth variant="outlined" component="label" startIcon={<UploadFile />}>
+            Upload {submission ? 'New' : ''} Anonymized File
+            <input
+              id="anonymizedFile"
+              name="anonymizedFile"
+              accept="application/pdf"
+              type="file"
+              hidden
+              onChange={(event: ChangeEvent<HTMLInputElement>) => {
+                if (event.target.files) {
+                  formik.setFieldValue("anonymizedFile", event.target.files[0]);
+                }
+              }}
+            />
+          </Button>
+        </Box>
+        {formik.touched.anonymizedFile && formik.errors.anonymizedFile && <FormHelperText>{formik.errors.anonymizedFile}</FormHelperText>}
+      </FormControl>
+      <FormControl fullWidth error={formik.touched.presentationFile && Boolean(formik.errors.presentationFile)}>
+        <Box sx={{ display: "flex", mt: 2 }}>
+          {formik.values.presentationFile &&
+            <Box sx={{ display: "flex", alignItems: "center" }}>
+              <FormHelperText>{formik.values.presentationFile?.name}</FormHelperText>
+              <IconButton onClick={() => { formik.setFieldValue("presentationFile", undefined) }}>
+                <ClearIcon />
+              </IconButton>
+            </Box>
+          }
+          <Button fullWidth variant="outlined" component="label" startIcon={<UploadFile />}>
+            Upload {submission ? 'New' : ''} Presentation File
+            <input
+              id="presentationFile"
+              name="presentationFile"
+              type="file"
+              hidden
+              onChange={(event: ChangeEvent<HTMLInputElement>) => {
+                if (event.target.files) {
+                  formik.setFieldValue("presentationFile", event.target.files[0]);
+                }
+              }}
+            />
+          </Button>
+        </Box>
+        {formik.touched.presentationFile && formik.errors.presentationFile && <FormHelperText>{formik.errors.presentationFile}</FormHelperText>}
+      </FormControl>
+      <FormControl fullWidth error={formik.touched.otherFiles && Boolean(formik.errors.otherFiles)}>
+        <Box sx={{ display: "flex", mt: 2 }}>
+          {formik.values.otherFiles &&
+            <Box sx={{ display: "flex", alignItems: "center" }}>
+              <Box sx={{width: "500px"}}>
+                {formik.values.otherFiles.map((file, index) => (
+                  <FormHelperText key={index}>{file.name}</FormHelperText>
+                ))}
+              </Box>
+              <IconButton onClick={() => { formik.setFieldValue("otherFiles", undefined) }}>
+                <ClearIcon />
+              </IconButton>
+            </Box>
+          }
+          <Button fullWidth variant="outlined" component="label" startIcon={<UploadFile />}>
```

Comment:
maybe we should set a max number (or max size) of files that can be uploaded here?